device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/buffers_defaults_t0.j2

```diff
@@ -1,7 +1,8 @@
 {% set default_cable = '5m' %}
-{% set ingress_lossless_pool_size =  '6687744' %}
+{% set ingress_lossless_pool_size =  '7719936' %}
+{% set ingress_lossless_pool_xoff =  '1032192' %}
 {% set egress_lossless_pool_size =  '13945824' %}
-{% set egress_lossy_pool_size =  '6687744' %}
+{% set egress_lossy_pool_size =  '7719936' %}
```

Comment:
What assumptions do you hold when calculating buffer values for different SKUs?

Reply:
Hi, Wei,
FYI.
For 3800 platform:
- C64: 32 100G down links and 32 100G up links.
- D112C8: 112 50G down links and 8 100G up links.
- D24C52: 24 50G down links, 20 100G down links, and 32 100G up links.
- D28C50: 28 50G down links, 18 100G down links, and 32 100G up links.

For 2700 platform:
- Mellanox-SN2700-C28D8 t0, assume 48 * 50G/5m + 8 * 100G/40m ports
- Mellanox-SN2700 (C32)
  - t0: 16 * 100G/5m + 16 * 100G/40m
  - t1: 16 * 100G/40m + 16 * 100G/300m